go/vt/automation/resharding_task_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package automation
+
+import (
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+)
+
+func TestReshardingTaskEmittedTasks(t *testing.T) {
+	reshardingTask := ReshardingTask{}
+
+	newTaskContainers, _, _ := reshardingTask.run(map[string]string{
+		"source_shard_rdonly_list": "rdonly1",
```

Comment:
It doesn't matter for the purpose of this test, but I think for documentation purposes, you should make the rdonly list read more like real tablet aliases.